client/web/src/user/settings/codeHosts/UserAddCodeHostsPage.tsx

```diff
@@ -327,6 +344,7 @@ export const UserAddCodeHostsPage: React.FunctionComponent<UserAddCodeHostsPageP
                                         onDidAdd={addNewService}
                                         onDidRemove={removeService(kind)}
                                         onDidError={handleError}
+                                        useGitHubApp={useGitHubApp && kind === ExternalServiceKind.GITHUB}
```

Comment:
This way the check will only be applied on github connection
```suggestion
                                        useGitHubApp={kind === ExternalServiceKind.GITHUB && useGitHubApp}
```

Reply:
Is there any benefit to doing it like this is this scenario? Not opposed to the change, just wondering what would make one check take preference... Actually as I was typing this it clicked in my head, if you order the `kind` check first, it will stop after that. But with `useGitHubApp` first, it will always have to do both checks.